packages/stream_chat/lib/src/api/channel.dart

```diff
@@ -507,17 +507,17 @@ class Channel {
 
   /// Pins provided message
   Future<UpdateMessageResponse> pinMessage(
-    Message message,
+    Message message, [
     Object? timeoutOrExpirationDate,
-  ) {
+  ]) {
     assert(() {
       if (timeoutOrExpirationDate is! DateTime &&
           timeoutOrExpirationDate != null &&
           timeoutOrExpirationDate is! num) {
         throw ArgumentError('Invalid timeout or Expiration date');
       }
       return true;
-    }(), 'Check for invalid token or expiration date');
+    }(), 'Check whether time out is valid');
```

Comment:
Can you correct this to 'timeout' instead of 'time out'?

Reply:
sal vatore

Reply:
lol ok